assets/scripts/views/actions.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * The floating actions
+ */
+export default class {
+  /**
+   * Initialize
+   */
+  constructor({
+    onMenuToggle = () => {},
+    onFiltersToggle = () => {},
+    onCenter = () => {}
+  }) {
```

Comment:
Aah, I see: These are the default options pointing to an empty handler using an object spread.